examples/Python/python-flask-webapp-sample/app.py

```diff
@@ -24,10 +25,12 @@ def callback():
     trim_prefixes=["testapp_settings_"],
     refresh_on=[WatchKey("sentinel")],
     on_refresh_success=callback,
+    feature_flag_enabled=True,
+    feature_flag_refresh_enabled=True,
 )
+feature_manager = FeatureManager(azure_app_config)
```

Comment:
Since we are not limiting the usage of the feature manager to kvs only from app config, does it make sense to create the feature manager from the flask config?
In this case, after line 32, we have:

```python
feature_manager = FeatureManager(app.config) 
```

Reply:
We can't if we want to support refresh.